docs/audits/gpt-errors-overall.md

```diff
@@ -0,0 +1,20 @@
+# Fix GPT errors
+
+## Overview
+
+This audit ensures that no warnings or errors related to Google Publisher Tags (GPT) are present. Reducing errorscan help 
+ensure that user experience is consistent and working as intended.
```

Comment:
"... and the ad tag setup is working as intended"
or
".. and the page is tagged as intended"

It sounds like you're saying "user experience is working as intended" which doesn't quite sound right to me.

Reply:
Agreed, adding the detail, thanks.